app/src/styles.scss

```diff
@@ -1,6 +1,6 @@
 /* You can add global styles to this file, and also import other style files */
-@import 'app/content-plugin/ui/application';
-@import 'app/content-plugin/ui/variables/font-family';
+@import 'projects/aca-content/src/lib/ui/application';
```

Comment:
this should be a reference to the NPM project? like we import ADF?

Reply:
moved to angular.json.

```
"styles": [
              ...
              "projects/aca-content/src/lib/ui/application.scss"
            ],
```

I think we are going to import it like from adf when we are going to install it in different repository, currently we are in the same repo (I am going to create a different PR in order to build it).